src/test/resources/com/puppycrawl/tools/checkstyle/checks/blocks/leftcurly/InputLeftCurlyHasLineBreakAfterWithComment.java

```diff
@@ -0,0 +1,42 @@
+/*
+LeftCurly
+option = eol
+tokens = (default)ANNOTATION_DEF, CLASS_DEF, CTOR_DEF, ENUM_CONSTANT_DEF, \
+         ENUM_DEF, INTERFACE_DEF, LAMBDA, LITERAL_CASE, LITERAL_CATCH, \
+         LITERAL_DEFAULT, LITERAL_DO, LITERAL_ELSE, LITERAL_FINALLY, LITERAL_FOR, \
+         LITERAL_IF, LITERAL_SWITCH, LITERAL_SYNCHRONIZED, LITERAL_TRY, LITERAL_WHILE, \
+         METHOD_DEF, OBJBLOCK, STATIC_INIT, RECORD_DEF, COMPACT_CTOR_DEF
+
+
+*/
+
+package com.puppycrawl.tools.checkstyle.checks.blocks.leftcurly;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.List;
+
+public class InputLeftCurlyHasLineBreakAfterWithComment {
+
+    void method() { // ok
```

Comment:
it is actually ok for in case config is EOL ?

we need to update doc to state that "comments after '{' are ignored" or something like this.

Reply:
Yes, Single line comments after left curly will be ignored if the config is EOL.  Please look at test case which contains Block comment in same file .

Like 
```
    void mehtod() { /* cmt */ int a; // violation ''{' at column 19 should have line break after'

    }

    void methi() { /* CheckStyle */     --->  ok  scenario
        int a;
    }
```

Reply:
please update doc, this should be explicitly explained to user.

Reply:
done